File: app/src/main/java/com/health/openworkout/core/workout/WallSit.java
package com.health.openworkout.core.workout;

import com.health.openworkout.R;
import com.health.openworkout.core.datatypes.WorkoutItem;

public class WallSit extends WorkoutItem {
    public WallSit() {
        super();
        setName(getContext().getString(R.string.workout_name_wall_sit));
        setDescription(getContext().getString(R.string.workout_description_wall_sit));
        setImagePath("wall_sit.png");
        setVideoPath("wall_sit.mp4");
    }
}

File: app/src/main/java/com/health/openworkout/core/workout/WallSit.kt
package com.health.openworkout.core.workout

import com.health.openworkout.R
import com.health.openworkout.core.datatypes.WorkoutItem

class WallSit : WorkoutItem() {
    init {
        name = context.getString(R.string.workout_name_wall_sit)
        description = context.getString(R.string.workout_description_wall_sit)
        imagePath = "wall_sit.png"
        videoPath = "wall_sit.mp4"
    }
}
